test/param.spec.js

```diff
@@ -65,6 +65,11 @@ describe("Testing Params", () => {
         enabled: false
       }
     })).to.equal(false);
+    expect(param.get({
+      body: {
+        enabled: null
+      }
+    })).to.equal(null);
```

Comment:
This was a case where the null check failed for `bool`, so I thought it best to add in test cases for the remaining parameters as well.